DocumentationValidator/Program.cs

```diff
@@ -14,11 +15,69 @@ class Program
         static async Task<int> Main(string[] args)
         {
             IList<NuGetLogCode> allLogCodes = GetNuGetLogCodes();
-            List<NuGetLogCode> undocumentedLogCodes = new();
             HttpClient httpClient = new();
+            var cred = args.Length > 0 ? args[0] : "123";
+            var client = new GitHubClient(new ProductHeaderValue("nuget-documentation-validator"))
+            {
+                Credentials = new Credentials(cred)
+            };
 
-            Console.WriteLine($"Checking {allLogCodes.Count} log codes for documentation.");
+            List<NuGetLogCode> undocumentedLogCodes = await GetUndocumentedLogCodes(allLogCodes, httpClient);
+            Console.WriteLine($"Undocumented Log Codes Count {undocumentedLogCodes.Count}");
+
+            Dictionary<NuGetLogCode, List<string>> logCodeToIssue =
+                await GetIssuesForLogCodeAsync(client, undocumentedLogCodes);
+            PrintIssues(logCodeToIssue);
+
+            bool shouldCreateIssues = false;
+            _ = args.Length >= 2 ? bool.TryParse(args[1], out shouldCreateIssues) : false;
```

Comment:
If you have a bit of experience with System.CommandLine, it's fairly quick and easy to use it to create a nicer CLI experience, and you get free help/usage messages, and type parsing/error messages.

But that's easy enough to add later.

Reply:
Yeah, I used it in https://www.nuget.org/packages/GithubHelperTool/. 
I just copied most of my code from there.

Reply:
FWIW, the CommandLineParser package is not the one I meant. I think it's the one that NuGet.CommandLine.XPlat uses? (and hasn't   But [System.CommandLine](https://www.nuget.org/packages/System.CommandLine/) is the one that the dotnet CLI uses, and that the dotnet cli team maintains.  But it doesn't matter, just an FYI.